modulus/datapipes/gnn/__init__.py

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 # from .vortex_shedding_dataset import VortexSheddingDataset
-from .ahmed_body_dataset import AhmedBodyDataset
+# from .ahmed_body_dataset import AhmedBodyDataset
```

Comment:
Is there a reason to keep these here as commented out?

Reply:
Not really. I'll just remove them